torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1885,6 +1967,13 @@ def _simplify_floor_div(self, expr):
             self.evaluate_expr(eq_expr)
         return self.simplify(expr)
 
+    def _verify_valid_range(self, symbol, valid_range):
+        if symbol not in self.var_to_val:
+            return
+        if has_hint(symbol) and self.size_hint(symbol) not in valid_range:
```

Comment:
If `symbol in self.var_to_val`, then the symbol MUST have a hint.

Reply:
But I am not sure why you don't just remove the first two lines of this function

Reply:
This was raising. I am not sure that invariant works correctly atm?

Reply:
Repro?